app/Ptr/PtrUpdateService.php

```diff
@@ -22,15 +22,22 @@ class PtrUpdateService extends UpdateService {
    */
   private $ptrs;
 
+    /**
+   * @var PtrValidateRdns
+   */
+  private $rdns;
+
   /**
    * PtrUpdateService constructor.
    *
    * @param PtrService    $ptr
    * @param PtrRepository $ptrs
+   * @param PtrValidateRdns $rdns
    */
-  public function boot(PtrService $ptr, PtrRepository $ptrs) {
+  public function boot(PtrService $ptr, PtrRepository $ptrs, PtrValidateRdns $rdns) {
     $this->ptr = $ptr;
     $this->ptrs = $ptrs;
+    $this->rdns = $rdns;
```

Comment:
rdns doesn't seem like a very accurate name for this variable, maybe ptrValidator?